grafana/README.md

```diff
@@ -0,0 +1,22 @@
+# Grafana
+
+This directory contains useful out-of-box dashboards.
+
+# Constraints
+
+Dashboards contain [summary](../grafana/dashboards/summary) and [single](../grafana/dashboards/single) dashboards.
+* summary: Aggregated dashboards for a cluster.
+* single: dashboards for a single resource(e.g. `node` or `pod`).
+
+**Notice**
+* summary dashboards use `alias` as the `cluster` template variable.
```

Comment:
`external_labels` is just a common pattern, your usage is also valid, the label then just comes from a different source, but ends up being similar at query time.